src/components/ProjectItemLanguages.tsx

```diff
@@ -26,10 +27,11 @@ const LanguagesList: React.FC<ProjectItemLanguagesProps> = ({ langs, selectedLan
         <Typography
           component="span"
           key={lang}
+          onClick={() => {handleLangClick(lang);}}
           sx={{
             height: isLarge ? '27px' : '20px',
             fontSize: isInListItem ? '14px' : '16px',
-            cursor: 'auto',
+            cursor: 'pointer',
```

Comment:
This isn't good for accessibility, but it isn't the main way of filtering, so doesn't matter as much